inz-demo/internetnz/Issues/nethuiTicket/description.xml

```diff
@@ -0,0 +1,123 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo INZ Nethui Ticket</en>
+		<nl>Demo INZ Nethui Ticket</nl>
+	</Name>
+	<ShortName>
+		<en>Demo INZ Nethui Ticket</en>
+		<nl></nl>
+	</ShortName>
+	<SchemeManager>inz-demo</SchemeManager>
+	<IssuerID>internetnz</IssuerID>
+	<CredentialID>nethuiTicket</CredentialID>
+	<Description>
+		<en>INZ Nethui Ticket</en>
+		<nl></nl>
+	</Description>
+	<IssueURL>
+		<en></en>
+		<nl></nl>
+	</IssueURL>
+	<Attributes>
+		<Attribute id="id">
+			<Name>
+				<en>Ticket number</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Ticket number</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="event">
+			<Name>
+				<en>Event</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Event</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="eventDate">
+			<Name>
+				<en>Event date</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Event date</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="ticketType">
+			<Name>
+				<en>Ticket type</en>
+				<nl></nl>
+			</Name>
+			<Description>
+				<en>Ticket type</en>
+				<nl></nl>
+			</Description>
+		</Attribute>
+		<Attribute id="fullName">
```

Comment:
Do we need a name on the ticket if we used a name from our Attributes to get the ticket issued?

Reply:
NetHui ticket website enables you to buy a ticket for someone else. I know this is not a blocker for anything, but, as the form requires it, I think that could make sense to store the name that was informed at the form, that could be the same or different from the full name of the buyer.

Reply:
I imagine that this is a more relaxed credential.

Reply:
See below comments